### ISSUE TITLE
Write task description for PlaziBot on Wikidata

### ISSUE THREAD

**Issue by Daniel-Mietchen** [Daniel-Mietchen/ideas#553]
e.g. 
for a given taxon mined from the literature into Plazi, search it on Wikidata and add Plazi treatment ID to Wikidata item about taxon.

**Comment by Daniel-Mietchen** [comment]
Pinging @punkish @myrmoteras.

**Comment by Daniel-Mietchen** [comment]
Example: From
http://tb.plazi.org/GgServer/html/03FB8670D547674BFF48FF575098FB09 ,
I just got
- the paper's DOI ($1), which I fed into [Source MD](https://tools.wmflabs.org/sourcemd/?), which provided the paper's metadata in a format ingestible by [QuickStatements](https://tools.wmflabs.org/quickstatements/#), through which I created [the Wikidata item (Q42561394) about the paper](https://www.wikidata.org/wiki/Q42561394)
- the treatment ID ($2), which was to be added to the [Wikidata item (Q42562568) about the species](https://www.wikidata.org/wiki/Q42562568). Since this did not exist, I first had to create it, which I [did](https://www.wikidata.org/w/index.php?title=Q42562568&oldid=587895161) manually with the typical default parameters for taxon items:
  - English label: $3 (in this case, "Loxosceles griffinae")
  - English description: "$4 of $5" (in this case, "species of spider")
  - P31:Q16521 (instance of: taxon)
  - P225:$3 (taxon name; in this case, "Loxosceles griffinae")
  - P105:$6 (taxon rank; in this case, species, i.e. Q7432)
  - P171:$7 (parent taxon; in this case, Loxosceles, i.e. Q1950866)
  - P1343:$8 (described by source, i.e. the item about the paper; in this case, Q42561394)

Once this was done, I could [add the Plazi ID](https://www.wikidata.org/w/index.php?title=Q42562568&type=revision&diff=587895346&oldid=587895161).

**Comment by myrmoteras** [comment]
my way to do it is to add the new taxon: 
http://treatment.plazi.org/id/03FB8670-D547-6748-FF48-FAFA563AF808

- search Loxosceles haddadi, whether this taxon exists in Wikidata, if not
- create new item
- language en
- Label: Loxosceles haddadi
- Description: species of spider
-  instance of: species
-  parent taxon: Loxosceles (recluse spider)
-  taxon name: Loxosceles haddadi
-  taxon rank: species
-  Plazi-ID: 03FB8670D5476748FF48FAFA563AF808
- 

This is why I stopped doing it. I can't do anything about the Zootaxa DOI issue. Ideally, Wikidata would have access to this http://tb.plazi.org/GgServer/rdf/03FB8670D5476748FF48FAFA563AF808 or for nomenclaturial issues this http://tb.plazi.org/GgServer/lodRdf/03FB8670D5476748FF48FAFA563AF808 and with that wikidata would have a lot more data

**Comment by myrmoteras** [comment]
Didn't you and guido go through this an created a set of quick statements already?

Technically, the issue is to find out whether the new species (taxon) is already in wikidata, whether the parent taxon exists, whether the paper is already in wikidata, and for all positives get the respective  property

Do you know Cristiana Sarasua: I met her a couple of time on events here in Switzerland about open data, and she mentioned that she might try to get a student to work on getting treatmentBank data into Wikidata. She just started a position at the Uni Zürich in a Citizen Science group.

**Comment by myrmoteras** [comment]
I guess, the other issue has been the social, for us uncontrollabled side to get this "botified"

glad you pick this up again - I think this is soemthing where we could supply a lot of data to wikidata, and at the same time wikidata could have a real impact - my be together with GBIF - how we deal with taxonomic names

**Comment by Daniel-Mietchen** [comment]
Yes, I went through this workflow with Guido. It hasn't reached a stage that would allow for complete automation, though, and now, we have tools like [WikidataIntegrator](https://github.com/SuLab/WikidataIntegrator) (WDI) and the WDI-based [Fatameh](https://phabricator.wikimedia.org/source/tool-fatameh/), which allows OAuth-based Wikidata editing in a way that can be scaled technically.

In terms of social scalability, I think it will still take some time to get _all_ of million-scale datasets in. For instance, we have around 9 million scholarly papers in Wikidata by now, but no concrete plans for doing the same for the over ca. 100 million other scholarly papers that have a DOI.

So the task would be to work with what is already in Wikidata in terms of species, papers and authors, and to bring these things together before importing new content at scale.

For an overview of the usage of P1992 (Plazi ID), see https://tools.wmflabs.org/sqid/#/view?id=P1992&lang=en .

Yes, I met Cristina Sarasua at WikidataCon last weekend, and she mentioned that she's in contact with you.

**Comment by myrmoteras** [comment]
I don't think we are (yet) the right partner to talk about large scale. But we are good a creating seeds and figure out how to get something to work, to prepare the semantics etc.

What might be a good starting point is to extend the current workflows we have in place that minimize the risk for adding items that are already in Wikidata. These are the journals that we process on the day they are published: Zootaxa. EJT, Zoosystema, RSZ, all the Pensoft journals. If we can do this for the new species, we then could expand for redescriptions in the same articles, add more detaill or more journals.

There is a (slim?) chance that we get funding to expand Plazi activities quiet a bit by expanding the processing range of journals. We should now this in the next couple of weeks.
